README.md

```diff
@@ -1,24 +1,24 @@
-# Answers Hitchhiker Theme
+# Answers (Search) Hitchhiker Theme
 
 <div>
   <a href='https://coveralls.io/github/yext/answers-hitchhiker-theme?branch=master'>
     <img src='https://coveralls.io/repos/github/yext/answers-hitchhiker-theme/badge.svg?branch=master' alt='Coverage Status' />
   </a>
 </div>
 
-A [Jambo](https://github.com/yext/jambo) theme for building Answers experiences.
+A [Jambo](https://github.com/yext/jambo) theme for building Search experiences.
 
-Additonal resources for integrating Answers can be found at https://hitchhikers.yext.com/.
+Additional resources for integrating Search can be found at https://hitchhikers.yext.com/.
 
-Need help? Ask a question in the [Hitchhiker's Community](https://hitchhikers.yext.com/community/c/answers).
+Need help? Ask a question in the [Hitchhiker's Community](https://hitchhikers.yext.com/community/c/searcg/20).
```

Comment:
There's a typo in the actual URL, so this is intentional (until the website team fixes the URL)